Step02/app/src/main/java/com/kwoolytech/step02/MainActivity.java

```diff
@@ -0,0 +1,26 @@
+package com.kwoolytech.step02;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.animation.AnimationUtils;
+import android.widget.FrameLayout;
+
+public class MainActivity extends AppCompatActivity {
+    private BaseballManager baseballManager;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        baseballManager = new BaseballManager(
+                                  MainActivity.this,
+                                  (FrameLayout)findViewById(R.id.frameLayout),
+                                  findViewById(R.id.imageViewGlove),
+                                  AnimationUtils.loadAnimation(MainActivity.this, R.anim.shake));
+
+        findViewById(R.id.buttonThrow).setOnClickListener(baseballManager.GetButtonOnClickListener());
+        findViewById(R.id.imageViewGlove).setOnDragListener(baseballManager.GetGloveOnDragListener());
+        findViewById(R.id.frameLayout).setOnDragListener(baseballManager.GetLayoutOnDragListener());
+    }
```

Comment:
@kwoolytech looks more easily readable. Activity is just setting a content view and required listeners which is common a task for activity. You actually used "Delegate" design principle for enhancing code readability and maintainability. Great job !